packages/storybook/.storybook/preview.js

```diff
@@ -1,10 +1,13 @@
 import '@department-of-veterans-affairs/formation/dist/formation.min.css';
 import './style.scss';
 import '@department-of-veterans-affairs/formation/dist/formation';
-import {withHTML} from '@whitespace/storybook-addon-html/react';
+import { withHTML } from '@whitespace/storybook-addon-html/react';
 
 import '@department-of-veterans-affairs/component-library/dist/main.css';
-import {applyPolyfills, defineCustomElements} from '@department-of-veterans-affairs/component-library';
+import {
+  applyPolyfills,
+  defineCustomElements,
+} from '@department-of-veterans-affairs/web-components/loader';
```

Comment:
Aliasing `@department-of-veterans-affairs/component-library` to `react-components/src` means that we can't use that same name to import functions from `web-components`.